webhook-service/lib/webhook_config_test.go

```diff
@@ -123,6 +123,9 @@ spec:
           method: POST
           payload: "some payload"
           options: "some options"
+          user:
```

Comment:
Hi @odubajDT , I have been retrieving over it again and again. I have already added my user part here, but still test case fails. Any clue?
```
User:lib.Usr{Key:"", Value:""}
```

Reply:
Sorry @odubajDT ! you might have missed it. LMK how I can go ahead.